ads/mediavine.md

```diff
@@ -34,3 +34,13 @@ For details on the configuration semantics, please contact [Mediavine](http://ww
 * `data-site` - The site's unique name this ad will be served on. This is the same name from your Mediavine script wrapper.
 
 Each site must be approved and signed up with [Mediavine](http://www.mediavine.com) prior to launch. The site name will be the same as name in the Mediavine script wrapper. The site name `amp-project` can be used for testing and will serve placeholder ads.
+
+## Consent and GDPR
+ 
+Mediavine has set the `consentHandlingOverride` config value to `true` meaning that EU traffic will be prompted with the `<amp-consent>` component. Ad personalization is affected based on the user's interaction with this component.
```

Comment:
This is not correct. User will still need to include the `data-block-on-consent` attribute and `<amp-consent>` component to enable to this feature. 

@lannka I think we should come up with a template documentation here. WDYT

Reply:
Agree, we can have a place to have a unified language for this. `amp-ad.md` might be a good place.

Reply:
sorry, it should be ads/README.md, given it's ad network facing documentation.

Reply:
or we can have create a documentation for `_ping_`, and refer to `_ping_.md`

Reply:
Should I wait for the maintainers to make a template before I make this doc change?

Reply:
@timtyrrell FYI, the template is there: https://github.com/ampproject/amphtml/blob/master/ads/_ping_.md#user-consent-integration